InnerEye/ML/configs/ssl/CIFAR_SSL_configs.py

```diff
@@ -15,7 +15,7 @@ class CIFAR10SimCLR(SSLContainer):
     def __init__(self) -> None:
         super().__init__(ssl_training_dataset_name=SSLDatasetName.CIFAR10,
                          linear_head_dataset_name=SSLDatasetName.CIFAR10,
-                         ssl_training_batch_size=512,
+                         ssl_training_batch_size=128,
```

Comment:
Can we add a comment here saying how many GPUs we use (and from this the effective batch size)?